app/controllers/halls_controller.rb

```diff
@@ -0,0 +1,52 @@
+class HallsController < ApplicationController
+
+    before_action :set_hall, only: [:show, :update, :destroy]
```

Comment:
Compare the indentation here with any other file in the repository committed before, for example: https://github.com/lukasz-ligizynski/Project/blob/main/config/application.rb . As you can spot your file indentation has 4 spaces, the other one has an indentation of 2 spaces. You should keep it consistent for many reasons, mostly to avoid "noise" in the repository.

Here you can find how to set it in VS Code: https://stackoverflow.com/questions/34174207/how-to-change-indentation-in-visual-studio-code